wgsl/index.bs

```diff
@@ -4490,7 +4476,7 @@ the test name.
 * v-0021: Cannot re-assign a constant.
 * v-0022: Global variables must have a storage class.
 * v-0023: Entry point functions accept no parameters.
-* v-0024: Entry point functions return void.
+* v-0024: Entry point functions do not return anything.
```

Comment:
This should be removed as it isn't correct anymore, yea? We return `out` storage class results through the entry point return. Or, maybe made specific to `Compute shader entry points do not return anything` ?

Reply:
It would be strange to have a rule specifically for compute here. I think it should be handled by a general rule that the outputs from entry points can only be within a defined set of outputs for the relevant stage.

Reply:
It's probably fine, these entries should all go away I think and get merged into the text of their respective section. The original Idea was to have a list of all the validations at the end but they've quickly got out of date with the rest of the spec/ fallen behind. So, merging their content up is probably the right way to go anyway.